spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Linear.scala

```diff
@@ -44,7 +45,9 @@ class Linear[T: ClassTag](
   inputSize: Int,
   outputSize: Int,
   private var initMethod: InitializationMethod = Default,
-  withBias: Boolean = true
+  withBias: Boolean = true,
+  wRegularizer: Regularizer[T] = null,
```

Comment:
The constructor list is too long. Can we move the new constructor parameter out? and nochange in javadoc?